src/app/components/common/FundingSummary/FundingSummary.tsx

```diff
@@ -67,9 +67,9 @@ const FundingSummary = (props: Props) => {
 
     const columnValues = {
       [FundingSummeryColumns.FUNDING_SUMMARY]: (value: string | number) => payments.find(payment => payment.id === value)?.name,
-      [FundingSummeryColumns.FEE]: (value: string | number) => `${paymentsValue(Number(value))} SSV`,
```

Comment:
Do we also change the amount in the tranasction to toFixed(2)? it should be consistent with UI. please make sure.